### DISCUSSION TITLE
Promoting Graves Ontology to encourage more contributors

### DISCUSSION THREAD

**Discussion by Stephen-Gates** [muninn/graves#38] [Ideas]
The intersection of people interested in Linked Data and Cemeteries is, I suspect, very small.  This discussion is to capture ideas on how to promote the Graves Ontology to encourage more contributors.

As a starter, publish Muninn/Graves on: 

- [BARTOC](http://bartoc.org) - The Basic Register of Thesauri, Ontologies & Classifications
- [Linked Open Vocabularies](https://lov.linkeddata.es/dataset/lov/) 
- [The Linked Open Data Cloud](https://lod-cloud.net/datasets)

Other suggestions welcome.

**Comment by rwarren2** [comment]
We'll resubmit to the Linked Open Vocabularies, the entire Muninn dataset / ontologies are in the [Linked Open Data Cloud](https://lod-cloud.net/dataset/muninn-world-war-i) already. I'll put together a submission for BARTOC.

**Comment by rwarren2** [comment]
[Muninn Graves is also on xenodo](https://zenodo.org/record/4673761), which is also a good place to store the data about the Toowong Cemetery in perpetuity.
